### PR TITLE
Update windows-autopilot-hybrid.md

### PR DESCRIPTION
**Pull request by msbemba**
Made changes to the navigation as the document just  states Profile type: Select Domain Join.

### CODE REVIEW

**Comment by msbemba** [comment]
issue#https://github.com/MicrosoftDocs/memdocs/issues/2422

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: memdocs/autopilot/windows-autopilot-hybrid.md [hunk 1]
```diff
@@ -250,7 +250,7 @@ It takes about 15 minutes for the device profile status to change from *Not assi
     - **Name**: Enter a descriptive name for the new profile.
     - **Description**: Enter a description for the profile.
     - **Platform**: Select **Windows 10 and later**.
-    - **Profile type**: Select **Templates**, choose the template name **Domain Join** and select **Create**.
+    - **Profile type**: Choose **Templates**, Select the template name **Domain Join** and click on **Create**.
```
Comment (JohanFreelancer9) [inline memdocs/autopilot/windows-autopilot-hybrid.md:253]:
```suggestion
    - **Profile type**: Select**Templates**, choose the template name **Domain Join**, and select **Create**.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by PRMerger-2** [comment]
@msbemba : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@greg-lindsay Please merge this PR. Thanks

**Comment by PRMerger8** [comment]
@dougeby : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Pull request merged**